docs/user/supported_devices.rst

```diff
@@ -300,6 +300,10 @@ ramips-mt7620
 
   - WT3020AD/F/H
 
+# Xiaomi
+
+  - Mi WiFi Mini
```

Comment:
Shouldn't this be `MiWiFi Mini` without the space?

Reply:
Depends on where you look. OpenWRT and most other sites say Mi Wifi with the Space. I guess because Mi is the sub brand and WiFi is the product type.

Reply:
The runtime Model string is MiWifi Mini, so i would go for this.

Yes, Xiaomis naming is confusing but in this case consistency is key IMHO

Reply:
Done